konan/konan.properties

```diff
@@ -697,7 +703,7 @@ clangNooptFlags.mingw_x64 = -O1
 linkerNoDebugFlags.mingw_x64 = -Wl,-S
 linkerDynamicFlags.mingw_x64 = -shared
 linkerKonanFlags.mingw_x64 =-static-libgcc -static-libstdc++ \
-  -Xclang -flto-visibility-public-std -Wl,--dynamicbase \
+  -Wl,--dynamicbase \
```

Comment:
Why do you find it obsolete?

Reply:
I left a link in a commit message: https://github.com/msys2/MINGW-packages/issues/5321

Reply:
Ok, makes sense.